sheetstorm/tests/test_integrity_generate_test_data.py

```diff
@@ -248,3 +250,62 @@ def test_that_passing_text_instead_of_number_for_projects_arguments_should_raise
     def test_that_passing_any_value_for_positional_superuser_argument_should_raise_error(self, test_value):
         with self.assertRaises(CommandError):
             management.call_command("generate_test_data", f"--superuser={test_value}")
+
+    @parameterized.expand([(1,), ("test_text",)])
+    def test_that_passing_any_value_for_positional_set_size_argument_should_raise_error(self, test_value):
```

Comment:
There should be also test that when you select one of sizes you should be able to chose second